docs/administrators/operational_procedures.rst

```diff
@@ -15,6 +15,10 @@ Project Release for Production
 
 This process describes the steps to prepare an inmanta project for production release.
 
+All packages should be hosted in an internal, curated python repository (like nexus or devpi) and the default pip
+config, which pulls packages from https://pypi.org/ should never be used. See the :ref:`specify_location_pip` section
+for more information on how to set the project's pip configuration.
```

Comment:
```suggestion
All packages should be hosted in an internal, curated python repository (like nexus or devpi). For security reasons, the default pip config, which pulls packages from https://pypi.org/ should never be used. See the :ref:`specify_location_pip` section
for more information on how to set the project's pip configuration.
```

Reply:
I would also add an explanation about the security implications (or link to external source), now it is a bit vague

Reply:
Added link to PEP 708